pkg/network/tracer/connection/tracer.go

```diff
@@ -215,10 +213,11 @@ func NewTracer(config *config.Config, bpfTelemetry *nettelemetry.EBPFTelemetry)
 		// load the kprobe tracer
 		log.Info("fentry tracer not supported, falling back to kprobe tracer")
 		var kprobeTracerType kprobe.TracerType
-		closeTracerFn, kprobeTracerType, err = kprobe.LoadTracer(config, m, mgrOptions, perfHandlerTCP)
+		m, closeTracerFn, kprobeTracerType, err = kprobe.LoadTracer(config, mgrOptions, perfHandlerTCP)
 		if err != nil {
 			return nil, err
 		}
+		m.DumpHandler = dumpMapsHandler
```

Comment:
Shouldn't this be set for when the fentry tracer is running?

Reply:
Yes, fixed